docs/en/observability/profiling-upgrade.asciidoc

```diff
@@ -9,14 +9,19 @@
 [[profiling-upgrade-process]]
 == Upgrade process
 
-To upgrade Universal Profiling, you need to:
+To upgrade from all versions earlier than 8.10 (GA), you need to:
 
 . Perform a stack upgrade in {ecloud}
 . Stop profiling data ingestion
-. Delete existing profiling data
+. Delete existing profiling data 
 . Setup Universal Profiling from scratch
 . Start profiling data ingestion
 
+To upgrade from version 8.10 or later, you need to:
+

```

Comment:
The first step here should be:

```
. Stop profiling data ingestion
```

Reply:
Why would we suggest to stop ingestion?
If our data model will always be backward compatible, there is no need to stop ingesting data, as the new replicas will push the new version (maybe with additional doc fileds?) while the others will keep sending the old data.

Are you saying that we have strict mapping that does not allow adding fields?

Reply:
I mentioned this because the last step that was mentioned was to start profiling data ingestion. So we either have to add this step or remove the step to start data ingestion.